mitmproxy/contentviews/__init__.py

```diff
@@ -173,11 +43,23 @@ def get_content_view(viewmode: View, data: bytes, **metadata):
 add(image.ViewImage())
 add(query.ViewQuery())
 
-if protobuf.ViewProtobuf.is_available():
+if protobuf.ViewProtobuf.is_available():  # pragma: no cover
```

Comment:
The view shells out to `protoc`, which needs to be on PATH (this is what this function checks).
Default install does not add it to the PATH. We should use the protobuf library directly, so there's no excuse for hiding missing coverage here.